prometheus-pushgateway.yaml

```diff
@@ -35,6 +35,26 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: prometheus-pushgateway-bitnami-compat
+    dependencies:
+      provides:
```

Comment:
nit: this is a noop since the package name is already `prometheus-pushgateway-bitnami-compat`